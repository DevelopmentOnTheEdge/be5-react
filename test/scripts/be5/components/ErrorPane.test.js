import React          from 'react';
import renderer       from 'react-test-renderer';
import ErrorPane      from '../../../../src/scripts/be5/components/ErrorPane';
import testData       from '../testData.json'


test('snapshot', () => {
  const component = renderer.create(
    <ErrorPane value={testData.jsonApiError} frontendParams={{documentName: 'test'}}/>
  );

  expect(component.toJSON()).toMatchSnapshot();
});
