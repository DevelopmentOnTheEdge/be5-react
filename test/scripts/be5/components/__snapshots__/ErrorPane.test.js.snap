// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<div
  className="errorPane"
>
  <div
    className="errorPane__error"
  >
    <h1
      className="errorPane__title"
    >
      404
       - 
      Query not found: testtable.Test 1D2
    </h1>
    <br />
    <pre
      className="errorPane__code"
      dangerouslySetInnerHTML={
        Object {
          "__html": "Be5Exception: Query not found: testtable.Test 1D2
 ",
        }
      }
    />
  </div>
</div>
`;
